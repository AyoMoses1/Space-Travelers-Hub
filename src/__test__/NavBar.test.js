import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavbarSpace from '../components/NavBar';

it('NavBar component test', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <NavbarSpace />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
