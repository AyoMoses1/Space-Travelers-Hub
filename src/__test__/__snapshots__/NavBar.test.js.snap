// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component test 1`] = `
<nav
  className="navbar navbar-expand-md navbar-light bg-light"
>
  <div
    className="container-fluid"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="logo"
        src="logo.png"
      />
    </a>
    <h2
      className="title"
    >
      Space Travelersâ€™ Hub
    </h2>
    <button
      aria-controls="navbarScroll"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-end navbar-collapse collapse"
      id="navbarScroll"
    >
      <div
        className="my-3 displayFlex navbar-nav navbar-nav-scroll"
        onKeyDown={[Function]}
        style={
          Object {
            "maxHeight": "112px",
          }
        }
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
        <a
          className=""
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
        <span>
          |
        </span>
        <a
          className=""
          href="/myProfile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </div>
    </div>
  </div>
</nav>
`;
